doc/src/base/arrays.md

```diff
@@ -136,6 +136,20 @@ Base.findnext(::Any, ::Integer)
 Base.findnext(::Function, ::Any, ::Integer)
 Base.findprev(::Any, ::Integer)
 Base.findprev(::Function, ::Any, ::Integer)
+Base.findmin(::AbstractArray)
```

Comment:
This method (and some others that you moved) work on e.g. tuples as well, but is this another docstring then?

Reply:
I added `findmin` and moved `findmin!`: the former indeed has other methods, but for the latter, I can only find a single method signature taking `(::AbstractArray, ::AbstractArray, ::AbstractArray)`. 
The methods I removed from `collections.md` (and then readded to `arrays.md`) only had `::AbstractArray`-type signature variants.
Thanks for finding time to review this!